include/ipr/interface

```diff
@@ -2312,7 +2313,7 @@ namespace ipr {
    // An alias is always initialized -- with what it is an alias for.
    // The type of an Alias expression is that of its initializer.
    struct Alias : Category<Category_code::Alias, Decl> {
-      const Type& type() const { return initializer().get().type(); }
+      const Type& type() const override { return initializer().get().type(); }
       const Region& lexical_region() const final { return home_region(); }
```

Comment:
This probably should be `final`

Reply:
This looks to have found a bug. This behavior is override in the impl by decl_rep::type.

Reply:
@GabrielDosReis what should be the resolution here. This can't be fixed without some messy changes to the impl.

Reply:
OK, let's make the merge and then fix it afterward.